v21.2/cluster-setup-troubleshooting.md

```diff
@@ -376,59 +384,79 @@ CockroachDB's built-in disk stall detection works as follows:
+
+If these issues remain unresolved, affected nodes will miss their liveness heartbeats, causing the cluster to lose nodes and eventually become unresponsive.
+
+**Solution:** To diagnose and resolve an excessive workload concurrency issue:
+
+- [Check for high CPU usage.](common-issues-to-monitor.html#cpu-usage)
+
+- [Check your workload concurrency](common-issues-to-monitor.html#workload-concurrency) and compare it to your provisioned CPU.
+
+  - {% include {{ page.version.version }}/prod-deployment/resolution-excessive-concurrency.md %}
+
+- [Check LSM health](common-issues-to-monitor.html#lsm-health), which can be affected over time by CPU starvation.
+
+  - {% include {{ page.version.version }}/prod-deployment/resolution-inverted-lsm.md %}
+
 ## Memory issues
 
 ### Suspected memory leak
 
-A CockroachDB node will grow to consume all of the memory allocated for its `cache`. The default size for the cache is ¼ of physical memory which can be substantial depending on your machine configuration. This growth will occur even if your cluster is otherwise idle due to the internal metrics that a CockroachDB cluster tracks. See the `--cache` flag in [`cockroach start`](cockroach-start.html#general).
+A CockroachDB node will grow to consume all of the memory allocated for its `cache`, [even if your cluster is idle](operational-faqs.html#why-is-memory-usage-increasing-despite-lack-of-traffic). The default cache size is 25% of physical memory, which can be substantial, depending on your machine configuration. For more information, see [Cache and SQL memory size](recommended-production-settings.html#cache-and-sql-memory-size).
 
 CockroachDB memory usage has the following components:
 
-- **Go allocated memory**: Memory allocated by the Go runtime to support query processing and various caches maintained in Go by CockroachDB. These caches are generally small in comparison to the RocksDB cache size. If Go allocated memory is larger than a few hundred megabytes, something concerning is going on.
-- **CGo allocated memory**: Memory allocated by the C/C++ libraries linked into CockroachDB and primarily concerns the block caches for the storage engine (this is true for both [Pebble (the default engine) and RocksDB](cockroach-start.html#storage-engine)). This is the “cache” mentioned in the note above. The size of CGo allocated memory is usually very close to the configured cache size.
-- **Overhead**: The process resident set size minus Go/CGo allocated memory.
+- **Go allocated memory**: Memory allocated by the Go runtime to support query processing and various caches maintained in Go by CockroachDB.
+- **CGo allocated memory**: Memory allocated by the C/C++ libraries linked into CockroachDB and primarily concerns the block caches for the [Pebble storage engine](cockroach-start.html#storage-engine)). This is the "cache" mentioned in the note above. The size of CGo allocated memory is usually very close to the configured cache size.
```

Comment:
Maybe avoid "above" as a direction? I wonder if this would be better as an anchor link? Because I'm not sure which note is meant here.